website/views.py

```diff
@@ -105,7 +105,11 @@ def post(self, request):
                 user.first_name = first_name
                 user.last_name = last_name
                 if len(request.FILES) != 0:
-                    profile.photo = photo
+                    if (photo.size <= 4 * 1024 * 1024):
```

Comment:
can we have this magic number in only 1 location as a constant?  Perhaps it makes sense to put into the site settings? Maybe also the max image size (500x500) as well?